lib/mix/lib/mix/tasks/deps.partition.ex

```diff
@@ -0,0 +1,261 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: 2021 The Elixir Team
+
+defmodule Mix.Tasks.Deps.Partition do
+  @moduledoc false
+  use Mix.Task
+
+  ## Server
+
+  @deps_partition_install_mix_exs ~c"deps.partition.mix.exs"
+
+  def server(deps, count, force?) do
+    {:ok, socket} = :gen_tcp.listen(0, [:binary, packet: :line, active: false])
+
+    try do
+      server(socket, deps, count, force?)
+    after
+      :gen_tcp.close(socket)
+    end
+  end
+
+  defp server(socket, deps, count, force?) do
+    elixir =
+      System.get_env("MIX_OS_DEPS_COMPILE_PARTITION_ELIXIR_EXECUTABLE") ||
```

Comment:
Should we document this one?
I was trying it out and it failed because my locally built mix was trying to use the asdf installed elixir.
Or I wondered, could we solve it by introducing something like `Mix.elixir_interpreter/0`?

Reply:
I decided to not expose it for now and wait if someone will have a use case for it. If they do, then we can just document it.